# BidArray.msg - Array of bids
std_msgs/Header header
decentralized_auction_mm/Bid[] bids